ui-tests/src/test/java/org/openmrs/reference/page/PatientVisitsDashboardPage.java

```diff
@@ -44,6 +48,22 @@ public PatientVisitsDashboardPage(Page parent, WebElement waitForStaleness) {
     public String getPageUrl() {
         return "coreapps/patientdashboard/patientDashboard.page";
     }
+    
+    public void goToEditVitalsPage() {
+        clickOn(EDIT_VITALS_ICON);
+    }
+    
+    public void goToViewVitalsPage() {
+    	clickOn(VIEW_VITALS_ICON);
+    }
+    
+    public void deleteVitals() {
+    	clickOn(DELETE_VITALS_ICON);
+    }
+    
+    public void confirmDeleteVitals() {
+    	clickOn(COMFIRM_DELETE_BUTTON);
```

Comment:
Worth looking into the indention to have something like;
```js
{
    public void goToViewVitalsPage() {
    	  clickOn(VIEW_VITALS_ICON);
    }

    public void deleteVitals() {
       	  clickOn(DELETE_VITALS_ICON);
    }

    public void confirmDeleteVitals() {
    	  clickOn(COMFIRM_DELETE_BUTTON);
      }
```

Reply:
@kdaud I don't really get you when you talk about indention in this sense.Where would that first opening brace be coming from, and doing what around those methods? Could you post the entire class?

Reply:
> @kdaud I don't really get you when you talk about indention

@Irenyak1 you are right !

Its the nasty of git the way it has stubbornly displayed the indention in those respective methods. Have just crossed the file behind the seen and affirmed the indention is alright!

Reply:
Alright @kdaud